Pollo/Views/Cards/NavigationTitleView.swift

```diff
@@ -36,9 +37,10 @@ class NavigationTitleView: UIView {
     let secondaryLabelHeight: CGFloat = 15
     let secondaryLabelTopOffset: CGFloat = 2
     let secondaryLabelWidth = UIScreen.main.bounds.width * 0.8
-
-    override init(frame: CGRect) {
+    
+    init(buttonWidth: CGFloat, frame: CGRect) {
```

Comment:
lets keep the frame as the first argument:
`init(frame: CGRect, buttonWidth: CGFloat)`